### ISSUE TITLE
Add gn-cloud-ogc-api-records-service

### ISSUE THREAD

**Issue by fvanderbiest** [georchestra/docker#145]
See https://github.com/etalab/data.gouv.fr/issues/913 for the motivation

**Comment by fvanderbiest** [comment]
For the record, here's what we're currently using for geo2france:
```
  ogc-api-records:
    image: georchestra/gn-cloud-ogc-api-records-service:2023-01-17-es7.15.1
    environment:
      LANG: en_US.UTF-8
      SERVER_SERVLET_CONTEXT_PATH: /ogc-api-records
      SPRING_CONFIG_LOCATION: file:///etc/georchestra/geonetwork/microservices/ogc-api-records/config.yaml
      SPRING_PROFILES_ACTIVE: standalone
    volumes:
    - georchestra_datadir:/etc/georchestra
 ```

https://hub.docker.com/r/georchestra/gn-cloud-ogc-api-records-service/tags
Looks like a manual build, since `2023-01-17-es7.15.1` is 2 months old, while `latest` was pushed a year ago.

The geonetwork project also publishes a [geonetwork/gn-cloud-ogc-api-records-service](https://hub.docker.com/r/geonetwork/gn-cloud-ogc-api-records-service/tags) image and provides a `4.2.3-0` tag which is quite recent (13 days). No `latest` unfortunately (but this should be pretty easy to add to the [workflow file](https://github.com/geonetwork/geonetwork-microservices/blob/main/.github/workflows/ci.yaml)).

We should probably give it a try.

**Comment by jahow** [comment]
closed by #172
